agent/util-scripts/pbench-tool-meister-start

```diff
@@ -446,11 +496,14 @@ def main(argv):
     # is the same on the local host as it is on any remote host.
```

Comment:
This comment really belongs above the previous line, if you're mucking with this code.